README.md

```diff
@@ -80,6 +81,32 @@ Allowed options:
 ./cli/OpenOrCadParser-cli --input file.OLB --verbosity 6 --keep >> file.txt
 ```
 
+## Generate Image from Library `Package`
+
+Using [pyorlib2ki](https://github.com/fjullien/pyorlib2ki) it is possible to import XML files into KiCad to generate for example images.
```

Comment:
s/for example/, for example,

Reply:
I reformulated this part. Thank you for checking.